portfolio/src/main/webapp/index.html

```diff
@@ -2,16 +2,93 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andres's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id="header" >Andres's Portfolio</h1>
+      <img src="/images/LinkedinProfile.jpg"/>
+      <h2>About Me</h2>
+      <p>
+        I have been obsessed with startups for as long as I could remember. 
+        Tech has the potential to do so much for so many people. I always 
+        have a side project I'm working on or an industry I'm looking into. 
+        I've failed many times but learn so much from each leap I take. My only 
+        goal is to help the world as much as I can through whichever companies 
+        I join or make.
+        <br>
```

Comment:
Instead of breaks, you can separate these by ending and restarting a `<p>` tag. `<p>` is meant to denote a paragraph so it's probably more appropriate to use that to separate each paragraph here.

`<p>My first paragraph</p>
<p>My second paragraph</p>` etc...
If you don't like the spacing, you can use CSS properties to move it around like `line-height`, `margin`, `padding`, etc.